CHANGELOG.md

```diff
@@ -25,11 +22,12 @@ Release date: TBD
 - Added an option to toggle the red dot icon for unread messages (default is on).
 
 #### Windows
+- Update Mattermost icon for desktop notifications in Windows 10.
 - Added an option to toogle the red dot icon for unread messages (default is on).
 
 
 ### Bug Fixes
-
+- Fixed but where window would restore the wrong state under some conditions.
```

Comment:
Proposed changelog entry: `Fixed an issue where the maximized state of the app window was lost in some cases.`

(I think it helps explain what we mean by the word "state")